keylime/tpm/tpm2.py

```diff
@@ -414,7 +414,7 @@ def __create_ek(self, asym_alg=None):
             outjson = config.yaml_to_dict(output)
             if outjson is not None and current_handle in outjson:
```

Comment:
I am also hitting an issue here, I suspect this is never executed otherwise we would find out.

```
Traceback (most recent call last):
  File "/usr/bin/keylime_agent", line 11, in <module>
    load_entry_point('keylime==0.0.0', 'console_scripts', 'keylime_agent')()
  File "/usr/lib/python3.6/site-packages/keylime/cmd/agent.py", line 15, in main
    keylime_agent.main()
  File "/usr/lib/python3.6/site-packages/keylime/keylime_agent.py", line 492, in main
    'cloud_agent', 'tpm_ownerpassword'))  # this tells initialize not to self activate the AIK
  File "/usr/lib/python3.6/site-packages/keylime/tpm/tpm2.py", line 935, in tpm_init
    self.__create_ek()
  File "/usr/lib/python3.6/site-packages/keylime/tpm/tpm2.py", line 416, in __create_ek
    if outjson is not None and current_handle in outjson:
TypeError: 'in <string>' requires string as left operand, not int
```

The handle is save as integer in the tpmdata.yml, but the persistent handles get from tpm2 tools are string, we probably should hex() it for the ``in`` test.